tests/invalid/Reference_Invalid_2.sysout

```diff
@@ -0,0 +1,3 @@
+../../tests/invalid/Reference_Invalid_2.whiley:0: invalid array expression
+
+^
```

Comment:
This error report is missing information. Shall I create a new issue to track the bug?